<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

<head>
    		<meta http-equiv="content-type" content="text/html;charset=utf-8" />
		<link rel='stylesheet' type='text/css' href='styles.css'/>
		<style type="text/css"></style>
		<link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,700, 700italic, 300italic,400italic,600' rel='stylesheet' type='text/css'>
		<link href='https://fonts.googleapis.com/css?family=Fira+Sans:400,500' rel='stylesheet' type='text/css'>
		<link rel="stylesheet" href="http://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.4.0/css/font-awesome.min.css">

	<!-- Code for Google Analytics Information -->
	<script>
	  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
	
	  ga('create', 'UA-71556839-1', 'auto');
	  ga('send', 'pageview');
	</script>
	<!-- End of Google Analytics information -->	




	</head>

	<title> Readings | PBoC @ GIST</title>

	<body >
<div id="container">
		<div id="header">
			<h1> PHYSICAL BIOLOGY of the CELL | GIST 2016
</h1>
		</div>
<div id="navbar">
		<li> <a href="index.html"><i class="fa fa-home "></i> Main </a></li>
		<li> <a href="people.html"><i class="fa fa-users "></i> People </a></li>
		<li> <a href="syllabus.html"><i class="fa fa-calendar "></i> Syllabus </a></li>
		<li> <a href="code.html"><i class="fa fa-code "></i> Code </a></li>
		<li class="current"> <i class="fa fa-briefcase "></i> Readings </li>
		<li> <a href="homework.html"><i class="fa fa-calculator "></i> Homework </a></li>
		<li> <a href="lab.html"><i class="fa fa-flask"></i> Lab </a></li>
		<li class="last"> <a href="http://rpgroup-pboc.github.io/GIST_Evolution_2016/index.html"><i class="fa fa-university "></i> Evolution @ GIST </a></li>
				<a href="http://caltech.edu"><img
       src="images/caltech.png" style="height: 35px"></a><br />

	<a href="http://gist.ac.kr"><img
       src="images/gist.png" style="width: 120px; padding-left: 5px"></a><br />

<br />
       <a href="https://github.com/RPGroup-PBoC/GIST_PBoC_2016"><span style="padding-left: 10px"><i class="fa fa-github-square"></i> git repo </span></a>
	</div>
	<div id="banner"> 
		<i class="fa fa-briefcase fa-inverse fa-1x"></i> files </div>


	<div id="content">
		<h1>Lecture Slides</h1>
		<hr />
		<p>Lecture slides will periodically added here throughout the
		course. These links will be password protected, so please do
		not share these slides outside of the course.</p>

		<div id="file-links">
			<li><a href="http://www.rpgroup.caltech.edu/courses/PBoC_GIST_2016/protected/ThinkingEcoli2016Post.pdf">Lectures 1 & 2|</a> Thinking up <i>E. coli</i>.</li>
			<li><a href="http://www.rpgroup.caltech.edu/courses/PBoC_GIST_2016/protected/PostingTimeScalesBiologyGIST2016.pdf">Lectures 2 & 3|</a> Biological time scales.</li>
			<li><a href="http://www.rpgroup.caltech.edu/courses/PBoC_GIST_2016/protected/PostingThinkingUpaFly2016.pdf">Lectures 3|</a> Thinking up a fly.</li>
		</div>
					
		<h1> Papers Mentioned During Class </h1>
		<hr />
		<br />
		<h2> Monday, January 11 </h2>
		<div id="file-links">
			<li><a href="http://www.rpgroup.caltech.edu/courses/PBoC_GIST_2016/HeinemannEcoliProteome2016.pdf">
			Schmidt et al., (2016), <i> The quantiative and condition-dependent <u> 
					Escherichia coli</u> proteome,</i> Nature Biotechnology 34(1), pp.104-110.</a> 
			<a href="http://www.rpgroup.caltech.edu/courses/PBoC_GIST_2016/HeinemannEcoliProteomeSI.pdf">(supplemental information) </a>
			and 
				<a href="http://www.rpgroup.caltech.edu/courses/PBoC_GIST_2016/Cell_2014_Li.pdf">
			Li et al., (2014), <i> Quantifying Absolute Protein Synthesis Rates Reveals Principles Underlying Allocation of Cellular Resources</u>,</i> Cell 157(3), pp.624–635.</a> <br/>
			These two papers represent the state of the art in measuring the number of proteins in E. coli.



			<li><a href="http://www.rpgroup.caltech.edu/courses/PBoC_GIST_2016/Science_2014_Chen.pdf">
			Chen et al., (2014), <i> Lattice light-sheet microscopy: Imaging molecules to embryos at high spatiotemporal resolution,</i> Science 346(6208), pp.1257998–1257998.</a> <br/>
			Paper on a powerful new imaging tool, the lattice light-sheet microscope. This was developed by the lab of Eric Betzig.



			<li><a href="http://www.rpgroup.caltech.edu/publications/phillips2015.pdf">
			Phillips, Rob, (2015), <i> Theory in Biology: Figure 1 or Figure 7?,</i> Trends in Cell Biology, 25(12), p723–729 </a> <br/>
			Related to Rob's comments on turning our thinking about biology into formal mathematical predictions and confronting that math with experiments that haven't been performed before to demonstrate our understanding.			



			<li><a href="http://www.rpgroup.caltech.edu/courses/PBoC_GIST_2016/Nature_2011_Savin.pdf">
			Savin et al., (2011), <i> On the growth and form of the gut,</i> Nature, 476, p57–62 </a> <br/>
			Prediction of gut structure using ideas from mechanics.			



			<li><a href="http://mitpress.mit.edu/books/street-fighting-mathematics">
			Mahajan, Sanjoy, (2010), <i> Street Fighting Mathematics,</i> MIT Press.</a> <br/>
			Great resource on educated guessing and problem solving. Free download!		
		</div>
		<br />
		<h2> Tuesday, January 12 </h2>


		<p>Check out Li et al. and Schimdt et al. above! Li et al. count the protein copy number in <i>E. coli </i> with seqeuncing (Protect-Seq). Schimdt et al. count the copy number with mass spectrometry. </p>
		

			<div id="file-links">
			<li><a href="http://www.rpgroup.caltech.edu/courses/PBoC_GIST_2016/Molecular_Microbiology_2012_Bakshi.pdf">
			Bakshi et al., (2012), <i> Superresolution imaging of ribosomes and RNA polymerase in live Escherichia coli cells,</i> Molecular Microbiology, 85(1), p21-38 </a> <br/>
			This paper applies recent advances in superresolution imagining to quantiate the spatial distributions of ribosomes and RNA polymerase in <i>E. coli</i>.

			<li><a href="http://www.rpgroup.caltech.edu/courses/PBoC_GIST_2016/Nat_Cell_Biol_2006_Lee.pdf">
			Lee et al., (2006), <i> DNA primase acts as a molecular brake in DNA replication,</i> Nature, 439(2), p621–624 </a> <br/>
			Single-molecule measurement of DNA replication. We used this data in class to estimate the DNA replication rate. 	

			<li><a href="http://www.rpgroup.caltech.edu/courses/PBoC_GIST_2016/Journal_of_Bacteriology_1991_S_L_Gotta.pdf">
			Gotta et al., (1991), <i> rRNA transcription rate in Escherichia coli,</i> J. Bacteriology, 173(20), p6647–6649 </a> <br/>
			Here the authors use electron microscopy to measure the transcription rate in <i> E. coli </i>. 	
		</div>
		<br />
		<h2> Wednesday, January 13 </h2>

			<div id="file-links">
			<li><a href="http://www.rpgroup.caltech.edu/courses/PBoC_GIST_2016/Curr._Biol._2013_Lucas.pdf">
			Lucaset al., (2013), <i> Live imaging of bicoid-dependent transcription in Drosophila embryos,</i> Current Biology, 23(21), p2135-2139 </a> and <a href="http://www.rpgroup.caltech.edu/courses/PBoC_GIST_2016/Current_Biology_2013_Garcia.pdf">
			Garcia et al., (2013), <i> Quantitative Imaging of Transcription in Living Drosophila Embryos Links Polymerase Activity to Patterning.,</i> Current Biology, 23(21), p2140-2145 </a>
			 <br/>
			Quantitative imaging of transcription in <i>Drosophila</i> embryos. Garcia et al. uses the clever MS2 technique that Rob spoke about.
		</div>

		<h1> Further reading</h1>
		<hr />
		<p>The papers provided here are meant to provide an entry point
		into the literature for going more deeply into various topics
		covered -in class.   These papers have been picked either
		because they provide interesting and provocative experimental
		measurements of particular biological phenomena or because they
		show how to go about constructing theoretical models in the
		physical biology spirit described in the course. The papers
		that of most direct relevance to what we will cover in class
		are linked on the "Syllabus" part of the website. </p>

		<h2> Biology by the numbers: </h2>
		<div id="file-links">
		<li> <a
	      	 href="http://www.rpgroup.caltech.edu/courses/PBoC_CSHL_2015/files_2015/articles/Milo%20Cell%20SnapShot%202010.pdf">Uri
       		Moran et al., (2010), <i> SnapShot: Key Numbers in Biology</i>, <b>
       		Cell</b> 141, pp.1262</a> <br/> List of key numbers in biology, such as
       		the quantity and size of cellular components and the rates of cellular
       		processes.
		</li>

		<li> <a href="http://www.rpgroup.caltech.edu/courses/PBoC_CSHL_2015/files_2015/articles/Milo%20PNAS%202009%20%20Feeling%20for%20the%20numbers%20in%20Biology.pdf"> Rob Phillips and Ron Milo (2009), <i> A feeling for the numbers in biology,</i> PNAS 106, pp. 21465-71. </a> <br />

	This paper describes the role of biological numeracy in thinking about a variety of problems.
		
		<li> <a href="http://www.rpgroup.caltech.edu/courses/PBoC_CSHL_2015/files_2015/articles/eddy%20bayesian%20methods.pdf">
		Sean Eddy (2004), <i> What is Bayesian statistics?</i> Nature Biotechnology 22, pp.  1177-178. </a><br />
		This paper gives a compact but beautiful example of the
		power of Bayesian methods for figuring out the
		probability ofsome hypothesis given the data.

		<li> <a href="http://www.rpgroup.caltech.edu/courses/PBoC_CSHL_2015/files_2015/articles/First_exposure_Garcia_2007.pdf">
				Hernan Garcia et al. (2007), <i> A First Exposure to Statistical Mechanics for Life Scientists</i>.</a><br />
		This paper is a brief introduction to ideas from statistical mechanics that can
		be used to analyze a variety of problems in biology.
		
		</div>
		<br />
		<h2> Regulatory Biology </h2>
		<div id="file-links">
		<li><a href="http://www.rpgroup.caltech.edu/courses/PBoC_CSHL_2015/files_2015/articles/Garcia%20PNAS%202011.pdf">
				Hernan Garcia and Rob Phillips (2011), <i>Quantitative dissection of the simple repression input-output function</i>, PNAS 108(29) pp. 12173-8.</a><br />
		This paper demonstrates that thermodynamic models can make
		quantitative predictions about the level of gene expression as
		a function of repressor copy number and operator strength.  
	
		<li><a href="http://www.rpgroup.caltech.edu/courses/PBoC_CSHL_2015/files_2015/articles/bintu%20Curr%20Op%20Gen%20Dev%20Models%202005.pdf">
				Lacramioara Bintu et al. (2005), <i> Transcriptional regulation by the numbers: models,</i> Current Opinions in Genetics and Development 15, pp. 116-24.</a><br />
		This paper outlines an approach to creating quantitative models
		of gene expression using thermodynamics of the binding of
		transcription factors and RNA polymerase to DNA.
		
		<li><a href="http://www.rpgroup.caltech.edu/courses/PBoC_CSHL_2015/files_2015/articles/Golding_Cell_2005.pdf">
				Ido Golding (2005), <i>Real-time kinetics of gene activity in individual bacteria,</i> Cell 123, pp. 1025-36.</a> <br />

		This paper clearly demonstrates that the process by which mRNA
		is produced in the E.coli cell is stochastic in nature. A
		surprising observation is that the mRNA distribution is not
		Poisson, characterized by bursts in mRNA production. To this
		day the source of the stochasticity remains a mystery.

		<li><a href="http://www.rpgroup.caltech.edu/courses/PBoC_CSHL_2015/files_2015/articles/BergSourjikPNAS.pdf">
		Victor Sourjik and Howard C. Berg (2002), <i>Receptor sensitivity in bacterial chemotaxis</i>, PNAS 99, 132-7</a> and 
		<a href="http://www.rpgroup.caltech.edu/courses/PBoC_CSHL_2015/files_2015/articles/SourjikBergPNAS-2002-Sourjik-12669-74.pdf">
		Victor Sourjik and Howard C. Berg (2002), <i> Binding of the
		</i> Escherichia coli </i> response regulator CheY to its
		target measured in vivo by fluorescende resonance energy
		transfer,</i> PNAS 99, pp. 12669-74.</a> <br />
		These two papers use the method of FRET to examine the relation
		between chemoattractant concentration and the chemical
		reactions within cells that control the frequency of tumbles.
		
		<li><a href="http://www.rpgroup.caltech.edu/courses/PBoC_CSHL_2015/files_2015/articles/keymer06.pdf">
				Juan Keymer et al. (2006), <i> Chemosensing in </i>Escherichia coli<i>: Two regimes of two-state receptors,</i> PNAS 103, pp. 1786-91.</a>
		and
			<a href="http://www.rpgroup.caltech.edu/courses/PBoC_CSHL_2015/files_2015/articles/Tu-PNAS-2005-Mello-17354-9.pdf">
		Bernardo A. Mello and Yuhai Tu (2005), <i>An allosteric model
			for heterogeneous receptor complexes: Understanding
			bacterial chemotaxis responses to multiple stimuli</i>,
		PNAS 102(48), pp. 1753-9.</a><br />
		Theory of chemotaxis: These two papers show how simple ideas
		from equilibrium statistical mechanics can be used to
		understand the chemotactic response of E. coli to different
		concentrations of chemoattractant.

	
		
	</div>
	<br />
		<h2> The Physics of Genome Management</h2>

		<div id="file-links">	
		<li><a href="http://www.rpgroup.caltech.edu/courses/PBoC_CSHL_2015/files_2015/articles/kornbergStryernar_nucleosome_positioning.pdf">
		Roger Kornberg and Lubert Stryer (1988), <i>Statistical distribrutions of nucleosomes: nonrandom locations by a stochastic mechanism,</i> Nucleic Acids Research 16, pp. 6677-90.</a><br />
		This paper shows how a simple model of excluded volume predicts how nucleosomes will be organized around promoters.


		<li><a href="http://www.rpgroup.caltech.edu/courses/PBoC_CSHL_2015/files_2015/articles/Widom_Segal_2006_Nature.pdf">
		Eran Segal et al. (2006), <i> A genomic code for nucleosome positioning,</i> Nature 442, pp. 772-8.</a><br />
		This paper describe work aimed at determining genome wide nucleosome positioning preferences.

		<li><a href="http://www.rpgroup.caltech.edu/courses/PBoC_CSHL_2015/files_2015/articles/Widom_Segal_2009_Nature.pdf">
		Noam Kaplan et al. (2009), <i> The DNA-encoded nucleosome organization of a eukaryotic genome,</i> Nature 458, 362-6.</a><br />
		This paper describe work aimed at determining genome wide nucleosome positioning preferences.

		<li><a href="http://www.rpgroup.caltech.edu/courses/PBoC_CSHL_2015/files_2015/articles/PNAS-2010-Wiggins.pdf">
		Paul Wiggins et al. (2010),<i> Strong intranucleoid interactions organize the </i>E. coli<i> chromosome into a nucleoid filament,</i> PNAS 107, pp. 4991-5.</a><br />
		This paper looks at the spatial organization of the genome in a cell.
		
		<li><a href="http://www.rpgroup.caltech.edu/courses/PBoC_CSHL_2015/files_2015/articles/Bustamante%20and%20Smith%20Nature%202001.pdf">
		Douglas Smith et al. (2001), <i>The bacteriophage Φ29 portal motor can package DNA gains a large internal force,</i> Nature 413, 748-52.</a><br />
		In this paper, optical tweezers are used to study the forces needed to package double-stranded DNA into a viral capsid.

		<li><a href="http://www.rpgroup.caltech.edu/courses/PBoC_CSHL_2015/files_2015/articles/Tsankov_PLoS_Biology_2010.pdf">
		Alexander Tsankov et al. (2010), <i> THe role of nucleosome positioning in the evolution of gene regulation,</i> PLoS Biology 8, e1000414.</a><br />
		This interesting paper examines the genome-wide nucleosome positions in
		12 different yeast species. This data provides an excellent jumping off
		point for models of nucleosome positioning.

		<li><a href="http://www.rpgroup.caltech.edu/courses/PBoC_CSHL_2015/files_2015/articles/PolachWidomJMB1995.pdf">
		K.J. Plach and J. Widom (1995), <i>Mechanism of protein access to specific DNA sequences in chromatin: A dynic equilibrium model for gene regulation,</i> JMB 254, 130-49.</a><br />
		This paper examines how different sites within nucleosomes
		grant access to DNA binding proteins and quantifies how this
		accessibility depends upon the depth of the sites of interest
		within the nucleosome.

		</div>

		<br />
		<h2>Pattern formation in biology:</h2>

		<div id="file-links">
		<li><a href="http://www.rpgroup.caltech.edu/courses/PBoC_CSHL_2015/files_2015/articles/gregor+al_05.pdf">
		Thomas Gregor et al. (2005), <i>Diffusion and scaling during early embryonic pattern formation,</i> PNAS 102, pp. 18403-7.</a><br />
		Quantitative analysis reveals that diffusion based mechanism
		cannot account for morphogen gradient scaling in early embryos
		across closely related fly species.

		</div>
		<br />
		<h2>Evolution</h2>

		<div id="file-links">
		<li><a href="http://www.rpgroup.caltech.edu/courses/PBoC_CSHL_2015/files_2015/articles/Dobzhansky_1973.pdf">
				Theodsius Dobzhansky (1973), <i>Nothing in biology makes sense except in the light of evolution</i>, The American Biology Teacher 3, 125-9.</a> <br/>
		This classic article makes a compelling case for the primacy of evolution in the study of biology.


	</div>

	</div>

	</div>

		</div>
	</body>
</html>
		
		
		
		
